tools/releases/dockerfiles/Dockerfile.kumactl

```diff
@@ -1,18 +1,29 @@
-FROM alpine:3.12
+FROM bitnami/kubectl:1.18.8
```

Comment:
Do you have any numbers of how the image size changes as a result of this?

Reply:
was `65.3MB`
is `211MB`

Reply:
Could we have a two-stage build and just copy in the `kubectl` binary? If it needs a debian base we could use [debian slim](https://hub.docker.com/layers/debian/library/debian/buster-slim/images/sha256-01b65c2928fed9427e59a679e287a75d98551ea2061cf03c61be0c7e1fc40fef), which is ~ 25 MB?

Reply:
That was actually a great suggestion which I missed because of spending too much time on this issue already. Thank you @austince: -)

Reply:
:)